### ISSUE TITLE
Pagination buttons also on top of lists

### ISSUE THREAD

**Issue by TheRittler** [coolcrowd/control-ui#16]
I needed to start many experiments by cloning the newest experiment, and every time I wanted to do this, I had to scroll down to the bottom of the page and click on next, this is a bit annoying. I think additional next/back buttons on top would be more comfortable.

**Comment by LeanderK** [comment]
aren't next/back buttons always at the bottom of the page?

**Comment by TheRittler** [comment]
no, look e.g. in stack overflow. I've seen this more than just at the bottom...
Example: http://stackoverflow.com/questions/244777/can-i-use-comments-inside-a-json-file?rq=1

**Comment by LeanderK** [comment]
but look here http://stackoverflow.com/questions/tagged/java

**Comment by TheRittler** [comment]
it doesn't really matter, if you can find it without :wink: i think, its more comfortable with next/back also at the top

**Comment by LeanderK** [comment]
google has it on the bottom, bing also

**Comment by kelunik** [comment]
That's just because of #17, no actual value usually. Just use the keyboard shortcuts. Press `?`.

**Comment by TheRittler** [comment]
I agree, if we solle #17, this Problem is not important anymore. @LeanderK you need the Buttons on top, if you wann to come to far behind Experiments and don't have to read the Experiments on the current page

**Comment by kelunik** [comment]
Better have a search than an additional button.

**Comment by TheRittler** [comment]
Isn't our id chronological? Can't we sort this way?

**Comment by kelunik** [comment]
Currently yes, see #17, that's OT here.

**Comment by TheRittler** [comment]
oh yes, false issue :speak_no_evil:
